modules/bitgo/test/v2/unit/internal/tssUtils/eddsa.ts

```diff
@@ -565,20 +692,55 @@ describe('TSS Utils:', async function () {
         {
           from: 'bitgo',
           to: 'user',
-          publicShare: bitgoKeyShare.yShares[1].y + bitgoKeyShare.yShares[1].chaincode,
+          publicShare: params.bitgoKeyShare.yShares[1].y + params.bitgoKeyShare.yShares[1].chaincode,
           privateShare: encryptedBitgoToUserMessage.toString(),
-          vssProof: bitgoKeyShare.yShares[1].v,
+          vssProof: params.bitgoKeyShare.yShares[1].v,
         },
         {
           from: 'bitgo',
           to: 'backup',
-          publicShare: bitgoKeyShare.yShares[2].y + bitgoKeyShare.yShares[2].chaincode,
+          publicShare: params.bitgoKeyShare.yShares[2].y + params.bitgoKeyShare.yShares[2].chaincode,
           privateShare: encryptedBitgoToBackupMessage.toString(),
-          vssProof: bitgoKeyShare.yShares[2].v,
+          vssProof: params.bitgoKeyShare.yShares[2].v,
         },
       ],
     };
 
+    const userKeyId = userGpgKeyActual.keyPacket.getFingerprint();
+    const bitgoToUserPublicShare = Buffer.from(
+      await sodium.crypto_scalarmult_ed25519_base_noclamp(Buffer.from(params.bitgoKeyShare.yShares[1].u, 'hex')),
+    ).toString('hex') + params.bitgoKeyShare.yShares[1].chaincode;
+    const bitgoToBackupPublicShare = Buffer.from(
+      await sodium.crypto_scalarmult_ed25519_base_noclamp(Buffer.from(params.bitgoKeyShare.yShares[2].u, 'hex')),
+    ).toString('hex') + params.bitgoKeyShare.yShares[2].chaincode;
+
+    bitgoKeychain.walletHSMGPGPublicKeySigs = await createWalletSignatures(
+      params.bitgoGpgKey.privateKey,
+      params.userGpgKey.publicKey,
+      params.backupGpgKey.publicKey,
+      [
+        { name: 'commonKeychain', value: bitgoCombined.pShare.y + bitgoCombined.pShare.chaincode },
+        { name: 'userKeyId', value: userKeyId },
+        { name: 'backupKeyId', value: userKeyId },
```

Comment:
your passing in userKeyId for backupkeyId, which should fail because on line 720 you are passing in `params.backupGpgKey.publicKey`

Reply:
I bet the tests were passing because we are still reusing the same gpg key for the user and backup when we create TSS wallets in the SDK. I updated the function to pass the backupKeyId though.